src/components/sets/Planets.js

```diff
@@ -0,0 +1,124 @@
+import React, {useState, useEffect} from 'react';
+import Pagination from '../Pagination';
+import {TableHeader, TableBody, Table} from '@patternfly/react-table';
+import gql from 'graphql-tag';
+import ApolloClient from 'apollo-boost';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Table/table';
+
+const Planets = () => {
+  const [planets, setPlanets] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [planetsPerPage, setPlanetsPerPage] = useState(5);
+  const client = new ApolloClient({
+    uri: 'https://swapi.graph.cool/',
+  });
+  const getAllPlanets = () => {
+    const all = gql`
+    query{
+      allPlanets{
+        name
+        rotationPeriod
+        orbitalPeriod
+        diameter
+        climate
+        gravity
+        terrain
+        population
+        surfaceWater
+      }
+    }    
+    `;
+    return all;
+  }
+  useEffect(() => {
+    const fetchPlanets = async () => {
+      setLoading(true);
+      await client.query({query: getAllPlanets()})
+      .then(result => {
+      const data = result.data.allPlanets;
+      setPlanets(data);
+      });
+      setLoading(false);
+    };
+
+    fetchPlanets();
+  }, []);
+  const indexOfLastFilm = currentPage * planetsPerPage;
+  const indexOfFirstFilm = indexOfLastFilm - planetsPerPage;
+  const currentPlanets = planets.slice(indexOfFirstFilm, indexOfLastFilm);
+  const columns = [
+  'Name',
+  'Rotation Period',
+  'Orbital Period',
+  'Diameter',
+  'Climate',
+  'Gravity',
+  'Terrain',
+  'Population',
+  'Surface Water'
+  ];
+  const rows = currentPlanets.map(planet => (
+    {cells: [
+      planet.name,
+      planet.rotationPeriod,
+      planet.orbitalPeriod,
+      planet.diameter,
+      JSON.stringify(planet.climate),
+      planet.gravity,
+      JSON.stringify(planet.terrain),
+      planet.population,
+      planet.surfaceWater
+  ]}
+  ))
+
+  const paginate = (pageNumber) => setCurrentPage(pageNumber);  
```

Comment:
you don't need `()` around `pageNumber`, here.
These are minor fixes its up-to you whether you want to do it or not. 👍